common/production_types.txt

```diff
@@ -477,35 +477,35 @@ stables = {
 artisinal_workshop = {
 	template = RGO_template_clerks
 	output_goods = artisinal_goods
-	value = 1
+	value = 3.9
 	farm = yes
 }
 
 coal_mine = {
 	template = RGO_template_labourers
 	output_goods = coal
-	value = 2.4
+	value = 7
 	mine = yes
 }
 
 cotton_plantation = {
 	template = RGO_template_farmers
 	output_goods = cotton
-	value = 2.2
+	value = 7.9
```

Comment:
I think this is a little high. Originally I was only putting Cotton in Harad, but now that it's been expanded to Khand, Mordor, Gondor and Rohan, the shortage of cotton can partially be resolved by putting more cotton RGOs around